backend/taskhub.go

```diff
@@ -50,7 +51,22 @@ func (w *taskHubWorker) Shutdown(ctx context.Context) error {
 	}
 
 	w.logger.Info("workers stopping and draining...")
-	w.orchestrationWorker.StopAndDrain()
-	w.activityWorker.StopAndDrain()
+	defer w.logger.Info("finished stopping and draining workers!")
+
+	wg := sync.WaitGroup{}
```

Comment:
The purpose of this change is to speed up the shutdown by shutting down the workers in parallel, correct?

Reply:
yes